lib/srv/desktop/rdp/rdpclient/src/rdpdr.rs

```diff
@@ -120,6 +122,18 @@ impl TeleportRdpdrBackend {
         }
     }
 
+    pub fn handle_tdp_sd_delete_response(
+        &mut self,
+        tdp_resp: SharedDirectoryDeleteResponse,
+    ) -> PduResult<()> {
+        if let Some(resp) = self.fs.handle_tdp_sd_delete_response(tdp_resp)? {
+            self.write_rdpdr(resp)
+        } else {
+            // Nothing to send back to the server
+            Ok(())
+        }
```

Comment:
```suggestion
         match self.fs.handle_tdp_sd_delete_response(tdp_resp)? {
            Some(resp) => self.write_rdpdr(resp),
            None => Ok(()), // Nothing to send back to the server
        }
```

Reply:
Opting to ignore this here because this pattern gets eliminated in https://github.com/gravitational/teleport/pull/34122:

https://github.com/gravitational/teleport/pull/34122/commits/48d57c6de6704d71e571d666c819a274eb2b3828#diff-649de9ed3b460c7d074083e126f424535b1c48bcd9b6a881fa278c66da107c94R95-R111